conf/cvoc/readme.md

```diff
@@ -0,0 +1,209 @@
+        "term-parent-uri": "",
+        "allow-free-text": "false",
+        "vocabs": {
+            "unesco": {
+                "vocabularyUri": "http://skos.um.es/unescothes/CS000",
+                "uriSpace": "http://skos.um.es/unescothes/"
+            },
+        },
+        "managed-fields": {
+            "vocabularyName": "cvocDemoVocabulary",
+            "termName": "cvocDemoTerm",
+            "vocabularyUri": "cvocDemoVocabularyURI"
+        },
+        "retrieval-filtering": {
+            "@context": {
+                "termName": "https://schema.org/name",
+                "vocabularyName": "https://dataverse.org/schema/vocabularyName",
+                "vocabularyUri": "https://dataverse.org/schema/vocabularyUri",
+                "lang": "@language",
+                "value": "@value"
+            },
+            "@id": {
+                "pattern": "{0}",
+                "params": ["@id"]
+            },
+            "termName": {
+                "pattern": "{0}",
+                "params": ["/graph/uri=@id/prefLabel"]
+            },
+            "vocabularyName": {
+                "pattern": "{0}",
+                "params": ["/graph/type=skos:ConceptScheme/prefLabel"]
+            },
+            "vocabularyUri": {
+                "pattern": "{0}",
+                "params": ["/graph/type=skos:ConceptScheme/uri"]
+            }
+        }
+    }
+
+This example adds the field
+* term-parent-uri - skosmos allows limiting queries to a subset of a vocabulary, This parameter allows that (I don't have any description of its format handy though)
+
+To handle multiple fields, this example changes:
+* field-name - the name of the parent field
+* term-uri-field - the name of the child field in which the termURI will be stored
+* managed-fields - a list of child field values that correspond with the information the skosmos script is able to supply, namely the term name, vocabular name and vocabulary uri. 
+
+When the skosmos Javascript runs on this type of field, it hides all child fields and displays and allows input for the term uri as it normally would. However, when a slection is made, the script populates the values for all 4 fields. This simplfies using a controlled vocabulary service with an existing field (although a termURI field is needed and would have to be added to use skosmos with the citation.keyword field.) One limitation of this approach is that the child fields for the term and vocabular name only store the value for the current language (versus the full set of translations). 
+
+#How to make a new Javascript for your Service/Vocabulary
+
+The skosmos.js and people.js scripts in this branch are good examples to start from. people.js is appropriate for single fields (it could be extendded to handle multiple fields but has not) and for services that provide a single vocabulary choice. skosmos.js is an example that handles both multiple vocabularies and managed fields.

```

Comment:
```suggestion
The skosmos.js and people.js scripts are good examples to start from. people.js is appropriate for single fields (it could be extended to handle multiple fields but has not) and for services that provide a single vocabulary choice. skosmos.js is an example that handles both multiple vocabularies and managed fields.
```